homework/06LessonSix.js

```diff
@@ -36,7 +40,10 @@ var anotherPoint;
 //
 // What is the difference between point (an object literal, or associative array)
 // and anotherPoint (a point object constructed from the Point class)?
-//
+//       Point is an object has one or more properties separated by a comma-,
```

Comment:
Nice thorough answer. Good job connecting the fact that `point` and `anotherPoint` are variables from earlier questions